flopy/mf6/utils/codegen/templates/macros.jinja

```diff
@@ -0,0 +1,21 @@
+{% macro init_vars(vars, alias=false, indent=0, skip=none) %}
+{%- for name, var in vars.items() if name not in skip %}
+{% set v = var|untag -%}
+{% set n = (name if alias else v.name)|safe_name -%}
+{{ ""|indent(indent, first=true) }}{{ n }}{%- if v.default is defined %}={{ v.default|value }}{%- endif -%},
```

Comment:
sadly I don't think so, I played with this for a while and `indent()` was the only way to get properly formatted code, since this macro is called from various levels of indentation.